app/ui/src/app/connections/create-page/create-page.component.scss

```diff
@@ -26,25 +25,24 @@
   flex: 0 0 auto;
 }
 
-/deep/ syndesis-connections-connection-basics,
-/deep/ syndesis-connections-configure-fields,
-/deep/ syndesis-connections-review {
-  position: relative;
-  flex: 1 1 auto;
-  overflow-y: scroll;
-  overflow-x: hidden;
-  margin-left: -20px;
-  padding-left: 20px;
-  // Dock the scrollbar to the right
-  margin-right: -20px;
-  padding-right: 20px;
-  height: 100%;
-}
+::ng-deep {
+  syndesis-connections-connection-basics,
+  syndesis-connections-configure-fields,
+  syndesis-connections-review {
+    position: relative;
+    flex: 1 1 auto;
+    margin-left: -20px;
+    padding-left: 20px;
+    margin-right: -20px;
+    padding-right: 20px;
+    @include syn-scrollable-body;
```

Comment:
It's usually better to include the `@import` statement at the beginning of the CSS block, so you can override its own CSS rules if required.